volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -1072,6 +1072,12 @@ def get_name(self, context, task):
             fname = "Anonymous Mapping"
         return fname
 
+    def get_name(self, context, task) -> Optional[str]:
```

Comment:
Yeah, there wasn't a return type defined, so theoretically you could make a case for it being ok, but I'd at least bump the PATCH number, so we know that changes were made and be able to differentiate if strange things start being reported please.  Hopefully this is in line to be moved to its own separate module in the same style as the ftrace stuff at some point...

Reply:
ok where do I bump the PATCH number for this for now?